IPs/RSKIP271.md

```diff
@@ -27,8 +27,21 @@ After receiving a peg-out request, and if no previous request is waiting in the
 
 If [RSKIP272](https://github.com/rsksmart/RSKIPs/blob/master/IPs/RSKIP272.md) is not activated, then the peg-out fees will be distributed evenly between all peg-out requests as it is today, and no change is required. 
 
-A new Bridge method `getEstimatedFeesForNextPegOutEvent()`is created so that users can estimate the peg-out fees. This method returns the fees of a peg-out transaction containing (N+2) outputs and 1.5 inputs, where N is the number of peg-outs requests waiting in the queue.  This method will be deprecated if RSKIP272 is activated.
+A new Bridge method `getEstimatedFeesForNextPegOutEvent()`is created so that users can estimate the peg-out fees. This method returns the fees of a peg-out transaction containing (N+2) outputs and 2 inputs, where N is the number of peg-outs requests waiting in the queue. This method will be deprecated if RSKIP272 is activated.
 
+### Peg-out Requests Count
+A new Bridge method `getQueuedPegoutsCount()` is created. This method returns the amount of peg-outs requests waiting in the queue to be released in the next peg-out transaction i.e. releaseRequestQueue size.
+
+### Peg-out Transaction Creation Block Number
+A new Bridge method `getNextPegoutCreationBlockNumber()` is created. This method returns the exact block number where the next peg-out transaction is expected to be created. This is calculated by adding the current block number(after a successful pegout) to a defined value of T=3 hours of RSK blocks i.e nextPegoutCreationBlockNumber = currentBlockNumber + 3 hours of RSK blocks.
```

Comment:
I think it's best if we mention the amount of RSK blocks instead of the 3 hours (since that is relative). You could add that it is approximately 3 hours given the current block mining time

Reply:
Done